dir/path_test.go

```diff
@@ -137,24 +137,13 @@ func TestPathManager_LocalKey(t *testing.T) {
 			NewRootedFS("/home/exampleuser/.config/notation/", nil),
 		),
 	}
-	localkeyPath := path.Localkey("key1", KeyExtension)
-	if localkeyPath != "/home/exampleuser/.config/notation/localkeys/key1"+KeyExtension {
+	keyPath, certPath := path.Localkey("key1")
+	if keyPath != "/home/exampleuser/.config/notation/localkeys/key1"+LocalKeyExtension {
 		t.Fatal("get Localkey() failed.")
 	}
-}
-
-func TestPathManager_LocalKeyFailed(t *testing.T) {
-	path := &PathManager{
-		UserConfigFS: NewUnionDirFS(
-			NewRootedFS("/home/exampleuser/.config/notation/", nil),
-		),
+	if certPath != "/home/exampleuser/.config/notation/localkeys/key1"+LocalCertificateExtension {
+		t.Fatal("get Localkey() failed.")
```

Comment:
Why the returned paths is not having extensions included?

Reply:
if certPath != "/home/exampleuser/.config/notation/localkeys/key1"+`LocalCertificateExtension` {
extension is here.

Reply:
In path.go, we are retrieving the key path with extension included, right?
```
	keyPath, err := p.UserConfigFS.GetPath(LocalKeysDir, name+LocalKeyExtension)
```

If that is the case, then the returned path should be `/home/exampleuser/.config/notation/localkeys/key1.key` (rather than `/home/exampleuser/.config/notation/localkeys/key1`). Isn't it?

Reply:
Yes, it is. So, what is your question? The unit test checks the path with extension.

> 	keyPath, certPath := path.Localkey("key1")
	if keyPath != "/home/exampleuser/.config/notation/localkeys/key1"+LocalKeyExtension {
		t.Fatal("get Localkey() failed.")
	}
	if certPath != "/home/exampleuser/.config/notation/localkeys/key1"+LocalCertificateExtension {
		t.Fatal("get Localkey() failed.")
	}